testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/AbstractFirstBrokerLoginTest.java

```diff
@@ -133,7 +134,7 @@ public void testLinkAccountByReauthenticationWithPassword() {
 
         loginPage.login("password");
         waitForAccountManagementTitle();
```

Comment:
Shouldn't this be updated to login to test app, and check that is opened rather the checking login page isn't current page